src/GeniusYield/TxBuilder/Class.hs

```diff
@@ -244,6 +244,14 @@ signTxBodyWithStakeImpl kM txBody = (\(pKey, sKey) -> signGYTxBody txBody $ GYSo
 class (GYTxMonad (TxMonadOf m), GYTxSpecialQueryMonad m) => GYTxGameMonad m where
     -- | Type of the supported 'GYTxMonad' instance that can participate within the "game".
     type TxMonadOf m = (r :: Type -> Type) | r -> m
+    -- TODO: Maybe introduce a user generation config type that this function can take?
```

Comment:
Yes, I think we should at a later point, something like `CreateUserConfig` inside `GeniusYield.Test.Privnet.Ctx`.